crmsh/ui_cluster.py

```diff
@@ -243,7 +243,7 @@ def looks_like_hostnames(lst):
         storage_group = optparse.OptionGroup(parser, "Storage configuration", "Options for configuring shared storage.")
         storage_group.add_option("-p", "--partition-device", dest="shared_device", metavar="DEVICE",
                                  help='Partition this shared storage device (only used in "storage" stage)')
-        storage_group.add_option("-s", "--sbd-device", dest="sbd_device", metavar="DEVICE",
+        storage_group.add_option("-s", "--sbd-device", dest="sbd_device", metavar="DEVICE", action="append",
```

Comment:
Could you improve the `help` output? Just add `use ";" as separator for multi path` or similar

Reply:
OK, changed